src/Config/casters.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+declare(strict_types=1);
+
+use Tempest\Mapper\CasterConfig;
+
+return new CasterConfig();
```

Comment:
I think it makes more sense to have one `mapperConfig` and have several arrays in that class.